oras/provider.py

```diff
@@ -568,7 +577,14 @@ def chunked_upload(
         :type layer: dict
         :param refresh_headers: if true, headers are refreshed
         :type refresh_headers: bool
+        :param chunk_size: chunk size in bytes
+        :type chunk_size: int
         """
+
+        default_chunk_size = 16777216
```

Comment:
Ditto here - this should be the default size, from defaults.py, in the function.